docs/faq.md

```diff
@@ -0,0 +1,68 @@
+# F.A.Q.
+
+Here are a couple of common questions about JupyterLab and the transition from
+the classic Notebook interface.
+
+## Is the classic notebook going away ?
+
+No, at least not yet, it will though get way less updates. There will be a
+transition period from Classic notebook to JupyterLab. Currently the notebook
+server you get when you install Jupyter comes with Classic Notebook
+preinstalled. JupyterLab is an extension. When you install JupyterLab, the
+notebook server runs both classic notebook and JupyterLab. 
+
+When JupyterLab has been widely adopted, the server will come with Lab by
+default, and the classic notebook will be an extension. 
+
+## What will happen to my notebook files ?
+
+JupyterLab still use the same file format. You can open the same notebook both
+in JupyterLab and classic notebook. Notebook will still work on nbviewer, be
+rendered on GitHub... etc. JupyterLab is just a new interface. 
+
+## I like the notebook interface, I do not want to migrate to JupyterLab.
```

Comment:
I would consider combining these this section and the one above since the answers are similar.